Linux_basics_srinath

```diff
@@ -0,0 +1,141 @@
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+"dummy.txt" 1 line, 21 characters
+esc + :wq is used to come out of the vi text editor.
+
+sudo command stands for SuperUser do, if a command needs to be done with administrative or root privileges sudo is used.
+srinath@srinath-VirtualBox:~/Desktop/Git_Local_Repository/dummy$ sudo apt-get update
+[sudo] password for srinath: 
+Hit:1 http://dl.google.com/linux/chrome/deb stable InRelease
+Hit:3 https://download.docker.com/linux/ubuntu focal InRelease                 
+Get:4 http://security.ubuntu.com/ubuntu focal-security InRelease [109 kB]      
+Ign:2 https://pkg.jenkins.io/debian-stable binary/ InRelease
+Hit:5 http://de.archive.ubuntu.com/ubuntu focal InRelease
+Get:6 http://de.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]     
+Hit:7 https://pkg.jenkins.io/debian-stable binary/ Release                     
+Get:8 http://de.archive.ubuntu.com/ubuntu focal-backports InRelease [101 kB]
+Get:9 http://security.ubuntu.com/ubuntu focal-security/main amd64 DEP-11 Metadata [24,3 kB]
+Get:10 http://security.ubuntu.com/ubuntu focal-security/universe amd64 DEP-11 Metadata [56,5 kB]
+Get:12 http://de.archive.ubuntu.com/ubuntu focal-updates/main i386 Packages [406 kB]
+Get:13 http://de.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [760 kB]
+Get:14 http://de.archive.ubuntu.com/ubuntu focal-updates/main Translation-en [187 kB]
+Get:15 http://de.archive.ubuntu.com/ubuntu focal-updates/main amd64 DEP-11 Metadata [264 kB]
+Get:16 http://de.archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [726 kB]
+Get:17 http://de.archive.ubuntu.com/ubuntu focal-updates/universe i386 Packages [536 kB]
+Get:18 http://de.archive.ubuntu.com/ubuntu focal-updates/universe Translation-en [147 kB]
+Get:19 http://de.archive.ubuntu.com/ubuntu focal-updates/universe amd64 DEP-11 Metadata [281 kB]
+Get:20 http://de.archive.ubuntu.com/ubuntu focal-updates/multiverse i386 Packages [4.592 B]
+Get:21 http://de.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [15,5 kB]
+Get:22 http://de.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 DEP-11 Metadata [2.468 B]
+Get:23 http://de.archive.ubuntu.com/ubuntu focal-backports/universe amd64 DEP-11 Metadata [1.768 B]
+Fetched 3.736 kB in 7s (509 kB/s)                                              
+Reading package lists... Done
+srinath@srinath-VirtualBox:~/Desktop/Git_Local_Repository/dummy$ 
+
```

Comment:
There were few more important commands which you have missed. Please try to do them too. If you need help regarding that, I have few commands in my repositary Basic-commands